Makefile

```diff
@@ -73,7 +73,16 @@ push-example:
 
 # Runs the tests
 test:
-	go test --cover ./...
+	echo "" > coverage.txt
+	for d in `go list ./... | grep -v vendor`; do \
```

Comment:
Should we be excluding additional folders, like `examples`, or should we have tests for them, too?

Also, is it possible to exclude generated code, like `*.pb.go`?

This list is likely to grow, so maybe there's a `.gitignore` equivalent for code coverage?